### PR TITLE
Added bootstrap config

### PR DESCRIPTION
**Pull request by rrigdon**
This PR incorporates content from https://github.com/conda/conda/issues/9337#issuecomment-542604108

### CODE REVIEW

File: docs/source/user-guide/tasks/use-conda-with-travis-ci.rst [hunk 1]
```diff
@@ -123,3 +122,35 @@ Travis CI with conda.
...
+     - "3.7"
+
+   install:
+     - wget https://repo.anaconda.com/pkgs/misc/conda-execs/conda-latest-linux-64.exe -O conda.exe
+     - chmod +x conda.exe
+     - export CONDA_ALWAYS_YES=1
+     - ./conda.exe create -p $HOME/miniconda python=$TRAVIS_PYTHON_VERSION conda conda-build pytest six pytest-cov pytest-mock
+     - export PATH="$HOME/miniconda/bin:$PATH"
+     - hash -r
+     - conda build conda.recipe --no-test
```
Comment (msarahan) [inline docs/source/user-guide/tasks/use-conda-with-travis-ci.rst]: These two lines are where people would have their own code to build their package

File: docs/source/user-guide/tasks/use-conda-with-travis-ci.rst [hunk 2]
```diff
@@ -123,3 +122,35 @@ Travis CI with conda.
...
+   language: python
+   python:
+     - "2.7"
+     - "3.7"
+
+   install:
+     - wget https://repo.anaconda.com/pkgs/misc/conda-execs/conda-latest-linux-64.exe -O conda.exe
+     - chmod +x conda.exe
+     - export CONDA_ALWAYS_YES=1
+     - ./conda.exe create -p $HOME/miniconda python=$TRAVIS_PYTHON_VERSION conda conda-build pytest six pytest-cov pytest-mock
```
Comment (msarahan) [inline docs/source/user-guide/tasks/use-conda-with-travis-ci.rst]: People should add their own dependencies here.  It's probably worth making note of that in a comment.

**Comment by rrigdon** [comment]
@msarahan - I made the edits but they need another look to make sure.

File: docs/source/user-guide/tasks/use-conda-with-travis-ci.rst [hunk 3]
```diff
@@ -123,3 +122,34 @@ Travis CI with conda.
 
 For an example project building conda packages on AppVeyor, see
 https://github.com/rmcgibbo/python-appveyor-conda-example.
+
+Bootstrap your environment
+==========================
+
+If you encounter the "RemoveError" error where ``conda update conda`` tries to
+uninstall setup tools, this is likely a metadata issue. To bootstrap your
+environment, use the standalone conda approach in your .travis.yml:
```
Comment (msarahan) [inline docs/source/user-guide/tasks/use-conda-with-travis-ci.rst]:
```suggestion
environment, use the standalone conda approach in your appveyor.yml:
```

File: docs/source/user-guide/tasks/use-conda-with-travis-ci.rst [hunk 4]
```diff
@@ -123,3 +122,34 @@ Travis CI with conda.
 
 For an example project building conda packages on AppVeyor, see
 https://github.com/rmcgibbo/python-appveyor-conda-example.
+
+Bootstrap your environment
+==========================
+
+If you encounter the "RemoveError" error where ``conda update conda`` tries to
```
Comment (msarahan) [inline docs/source/user-guide/tasks/use-conda-with-travis-ci.rst]: Let's remove the whole "RemoveError" thing.  This approach is just plain better and should be the recommended way.

**Pull request merged**